### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by omar-labana**
Highlights on this pull request:
- Prepare the directory structure
- Edit folder and file structure
- Implement the App component
- Implement Button, Display and ButtonPanel components

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        text: '0',
+      },
+      {
+        text: '.',
+      },
+      {
+        text: '=',
+      },
+    ],
+  ];
```
Comment (mahmoud717) [inline src/components/ButtonPanel.js]: Change this to a normal 2d array without objects.

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @omar-labana**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] Please make sure that you having passing linters, for now, your es linters check is failing. Make sure to fix it.

- [x]  In this milestone, you were to integrate the Redux store into your app. Please make sure to do so. 

- [x] in the button panel component, the Buttons array could have been just a 2d array like this 
```javascript
const buttons = [[1,2,3, +], [4,5,6,-], ...etc]
```
this way it looks cleaner and easier to use. 
 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by omar-labana** [comment]
Hey @mahmoud717 no redux is mentioned in any of the requirements.  I am leaving this big array to add more stuff like tab index and hover classes after I am finished with it.

**Comment by mahmoud717** [comment]
hey there, @omar-labana 
I'm sorry about this, this is a mistake from my side. I was looking at the requirements for a different project. 

for now, just fix the linter issues and request another review.

**Comment by omar-labana** [comment]
@mahmoud717 I did, Thanks

**Review by whiz25 — APPROVED** [review]
## approved 👍🏿 👍🏿 👍🏿 
Nice work fixing linter errors as requested. Merge and proceed to the next milestone.


### Martin Afani
#### Slack: whiz25

**Pull request merged**